k8s/install/terraform/rke.tf

```diff
@@ -0,0 +1,547 @@
+resource "random_uuid" "random_cluster_id" {}
```

Comment:
After finishing the exec on this, the kubeconfig that comes out contains ip 127.0.0.1 instead of the elastic ip of the load balancer